request-server_test.go

```diff
@@ -439,9 +439,9 @@ func TestRequestStat(t *testing.T) {
 	require.NoError(t, err)
 	fi, err := p.cli.Stat("/foo")
 	require.NoError(t, err)
-	assert.Equal(t, fi.Name(), "foo")
-	assert.Equal(t, fi.Size(), int64(5))
-	assert.Equal(t, fi.Mode(), os.FileMode(0644))
+	assert.Equal(t, "foo", fi.Name())
+	assert.Equal(t, int64(5), fi.Size())
+	assert.Equal(t, os.FileMode(0644), fi.Mode())
```

Comment:
So, I accidentally broke these permissions tests, and the report kept saying that it got `0644` but wanted `80644` or something which was super weird because we were expected `0644`, then I realized the arguments were out of order. I haven’t evaluated any of the other tests. That would be a lot more work than this refactor needs. (We all know it doesn’t need more lines changed!)